src/amo/components/AddonReviewListItem/index.js

```diff
@@ -234,6 +254,24 @@ export class AddonReviewListItemBase extends React.Component<InternalProps> {
                 ? i18n.gettext('Edit my reply')
                 : i18n.gettext('Edit my review')}
             </a>
+            <ConfirmButton
```

Comment:
Thanks for adding this. It will help. Here are some small suggestions that I think would make it look better.

- Make it say *Delete* to be clear
- Use a proper cancel button to differentiate the word *Cancel* from the text prompt
- Always render the buttons in a row, even on mobile. I think the default of switching to column mode on mobile was more for the case of cancel text (not a button).

<img width="986" alt="screenshot 2018-08-28 15 00 13" src="https://user-images.githubusercontent.com/55398/44747989-3f80e280-aad4-11e8-8801-5d30750e041b.png">
<img width="714" alt="screenshot 2018-08-28 15 00 32" src="https://user-images.githubusercontent.com/55398/44747992-40b20f80-aad4-11e8-81e3-9f28674b397e.png">
<img width="322" alt="screenshot 2018-08-28 15 07 12" src="https://user-images.githubusercontent.com/55398/44747999-427bd300-aad4-11e8-9182-55df8c8d402a.png">

```diff
diff --git a/src/amo/components/AddonReviewListItem/index.js b/src/amo/components/AddonReviewListItem/index.js
index 46607cc3a..18a10c8dd 100644
--- a/src/amo/components/AddonReviewListItem/index.js
+++ b/src/amo/components/AddonReviewListItem/index.js
@@ -255,6 +255,8 @@ export class AddonReviewListItemBase extends React.Component<InternalProps> {
                 : i18n.gettext('Edit my review')}
             </a>
             <ConfirmButton
+              cancelButtonType="neutral"
+              confirmButtonText={i18n.gettext('Delete')}
               buttonType="cancel"
               className={makeClassName(
                 'AddonReviewListItem-control',
diff --git a/src/amo/components/AddonReviewListItem/styles.scss b/src/amo/components/AddonReviewListItem/styles.scss
index e6b0da742..4eb99974a 100644
--- a/src/amo/components/AddonReviewListItem/styles.scss
+++ b/src/amo/components/AddonReviewListItem/styles.scss
@@ -41,6 +41,10 @@
     align-items: flex-start;
     color: $blue-50;
   }
+
+  .ConfirmButton-buttons {
+    flex-flow: row-reverse;
+  }
 }
 
 .AddonReviewListItem-control {
```

Reply:
If we move this button/confirm button to one side (right side for instance), that might look better and might be easier to style as we could add some `margin-start` and/or a thin vertical border.

Reply:
What do you think of @willdurand's suggestion @kumar303? I agree that the confirm dialog might look better if it were all the way on the left, but I'm not sure we want to move the "Edit" link to the right of the "Flag" link.

Reply:
I'm not sure what this means. The confirm button is already on the right side. I think it would be good to land this patch so that people can try it on dev then follow up with further UX changes -- e.g. Phil may have some ideas for improvement, too.